packages/catalog-model/examples/spotify-api.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: spotify
+  description: The Spotify web API
+  tags:
+    - spotify
+    - rest
+  annotations:
+    backstage.io/definition-at-location: url:https://raw.githubusercontent.com/APIs-guru/openapi-directory/master/APIs/spotify.com/v1/swagger.yaml
```

Comment:
Yet another reason to get rid of location types :grin:

Reply:
What you mean by removing the types? Is there an issue were this is already discussed?

Reply:
Not sure if there's an issue tbh xP could create one. But either way it would be to use the protocol + hostname for selecting the location processor.

Reply:
Discussed it here: https://discordapp.com/channels/687207715902193673/687207715902193679/752511516988801054

Reply:
If I think about it from an end user perspective, we should get rid of it asap 😆 I will continue with this first and once that change has landed (or lands), we can fix it here, too.